### ISSUE TITLE
Documentation links to bad page

### ISSUE THREAD

**Issue by brennan-nc** [icambron/twix.js#108]
The documentation link, linked to in the readme, here:
![screen shot 2018-08-29 at 3 48 30 pm](https://user-images.githubusercontent.com/6774304/44817570-0794a080-aba3-11e8-94ae-94431bfe13b6.png)

goes to this page: http://isaaccambron.com/twix.js/docs.html which appears to be broken:

![2018-08-29 at 3 44 pm](https://user-images.githubusercontent.com/6774304/44817523-e6cc4b00-aba2-11e8-84f8-289748282651.png)

**Comment by icambron** [comment]
@brennan-nc That link works fine for me. It's not secure because at the time I didn't have a way to make github pages work with SSL. I'll switch it to use SSL (that works now) and you should see if someone's hacked your connection.

**Comment by icambron** [comment]
Done

**Comment by brennan-nc** [comment]
That's really strange. Are you sure the domain is working / the domain name hasn't expired? I've tried accessing it from my iPhone on a cell network and my mac on my home network and work network. Going to this URL: `http://isaaccambron.com/twix.js/` shows me a 404 page, no matter where I look at it from. That was not always the case.
![2018-09-06 at 9 03 pm](https://user-images.githubusercontent.com/6774304/45196232-7bc0eb00-b218-11e8-8a76-fa12e0bc48aa.png)

I'm not ruling out a problem on my end, but I've never encountered an issue quite like this.

I've tried hitting the IP directly as well, but it looks like the site's behind Cloudflare, so that doesn't work.

This isn't a huge deal for me, since I can access the site via web archive, but I do think something is wrong with the site config.

Anyways, I'll leave this to you, but I thought I'd share, in case this helps you get to the bottom of the problem, if there is indeed an issue on your end.
